### DISCUSSION TITLE
Where are the functional tests?

### DISCUSSION THREAD

**Discussion by cknaap** [ucum-org/ucum#321] [Q&A]
Hello, 

In our implementation of UCUM canonicalization, we historically have a file `UcumFunctionalTests.2..xml`. I actually wanted to raise an issue about one of the cases therein.
However, currently, I see on the site some documentation on functional tests [here](https://ucum.org/docs/functional-tests), but I cannot find the xml file with the actual tests anywhere. Also not here in the GitHub repo. Where has it gone? 
-- Christiaan

**Comment by grahamegrieve** [comment]
oh just found this, Christiaan. Here: https://github.com/FHIR/Ucum-java/tree/master/src/test/resources
